docs/releases.md

```diff
@@ -0,0 +1,127 @@
+
+# Release Documentation
+
+# Abstract
```

Comment:
I think this should be ## ?

Reply:
Fixed this one as well as other places with headings shifted a wrong way. Thanks